src/Native/Runtime/coreclr/gcinfodecoder.cpp

```diff
@@ -1476,7 +1476,7 @@ OBJECTREF* GcInfoDecoder::GetRegisterSlot(
     _ASSERTE(regNum >= 0 && regNum <= 14);
     _ASSERTE(regNum != 13);  // sp
 
-    DWORD **ppReg;
+    PTR_UIntNative *ppReg;
```

Comment:
Could you please make this change under `FEATURE_REDHAWK` - similar how it is done for AMD64?